spk/bazarr/src/service-setup.sh

```diff
@@ -0,0 +1,33 @@
+# Package
+PACKAGE="bazarr"
+
+# Others
+PYTHON_DIR="/usr/local/python3"
+
+
+VIRTUALENV="${PYTHON_DIR}/bin/virtualenv"
+VAR_DIR="${SYNOPKG_PKGDEST}/var"
+
+SC_USER="sc-${PACKAGE}"
+GROUP="sc-download"
+SVC_BACKGROUND=y
+PID_FILE="${VAR_DIR}/bazarr.pid"
+SVC_WRITE_PID=y
+SVC_CWD="${SYNOPKG_PKGDEST}/share/${PACKAGE}"
+
+service_postinst ()
+{
+
+    # Create a Python virtualenv
+    ${VIRTUALENV} --system-site-packages ${SYNOPKG_PKGDEST}/env >> ${INST_LOG} 2>&1
+
+    # Install the wheels (using virtual env through PATH)
+    ${SYNOPKG_PKGDEST}/env/bin/pip install --no-deps --no-index -U --force-reinstall \
```

Comment:
If wheels are too large, you may reuse generic alternate TMP location support from @hgy59